cadcdata/cadcdata/tests/test_core.py

```diff
@@ -287,17 +285,17 @@ def test_help(self):
 
         # get -h
         usage = \
-'''usage: cadc-data get [-h] [-V]
+'''usage: cadc-data get [-h]
                      [--cert CERT | -n | --netrc-file NETRC_FILE | -u USER]
                      [--host HOST] [--resourceID RESOURCEID] [-d | -q | -v] -a
                      ARCHIVE [-o OUTPUT] [--cutout CUTOUT] [-de] [--wcs]
                      [--fhead]
-                     fileID [fileID ...]
+                     fileName [fileName ...]
```

Comment:
The definition of the filename in the help is: The name of the file in the archive, which in my opinion is the correct one. The file name in the content disposition is the name of the file saved on the local system. It is a bit ambiguous but that's because the data WS does that magic of finding the file whether you use extensions or not (most of the time). Otherwise I would have probably said that the filename is the name of the file that was uploaded to the archive, that is if the file was blah.fits.gz then that's the name of the file you have to look for.